src/routes/auth.js

```diff
@@ -124,14 +125,36 @@ router.post(
 			currentRefreshToken
 		)
 
-		res.status(200).json({
+		res.status(201).json({
```

Comment:
@beblicarl, not in the scope of this issue and also not a fundamental resource the app deals with, its a means to an end so 200 is fine

Reply:
thank you